### DISCUSSION TITLE
Create RDS postgres instance from snapshot

### DISCUSSION THREAD

**Discussion by fegmorte** [aws/copilot-cli#3971] [Q&A] [answered]
Hi everyone,

Is there a way to create a RDS instance from a snapshot in aws copilot ?
I found how to create a storage RDS but no option to load a snapshot.

I will appreciate your help

Fred

**Comment by paragbhingre** [comment, marked as answer]
@fegmorte I think you can utilize Copilot's addons feature to write your Cloudformation template for `DBInstance` and specify `DBSnapshotIdentifier` inside it to create an RDS instance from a snapshot.
I have not tried this by myself, but this is one way I can think of for now.

**Reply by fegmorte** [reply]
Hello @paragbhingre I just found that in the documentation yesterday and i have started to work on.
Thank you for your reply
